modules/ROOT/pages/manage-monitor-applications.adoc

```diff
@@ -15,11 +15,16 @@ By default, Runtime Fabric is configured to send application metrics to Anypoint
 
 If you need an outbound proxy, configure Runtime Fabric to use a SOCKS5 proxy by following the instructions in xref:manage-proxy.adoc[Manage Proxies Used by Runtime Fabric].
 
-To verify outbound network connectivity, ensure that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 5044 to:
+To verify outbound network connectivity, ensure that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 443 to:
 
 . US control plane: `dias-ingestor-nginx.prod.cloudhub.io`
```

Comment:
Should update to "dias-ingestor-router.us-east-1.prod.cloudhub.io" for agent version 1.8.50 and later

Reply:
Same with below (if necessary).

Reply:
Updated hostnames.